docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -87,11 +90,13 @@ Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your
 
```

Comment:
Can't comment on line 63 but "Analog to Digital Converter" should become "analog-to-digital converter," and I think "small oval" -> "half circle"